kybra/compiler/kybra_generate/src/method_utils/params.rs

```diff
@@ -50,15 +65,15 @@ impl SourceMapped<&Located<StmtKind>> {
                     }
                 };
 
-                Ok(crate::errors::collect_kybra_results(param_results)?)
+                param_results.into_iter().collect_results()
```

Comment:
I'm not fully understanding this line. Can you explain it to us on a call?

Reply:
done